### ISSUE TITLE
[Feature Request] Modded Ore Support

### ISSUE THREAD

**Issue by MokahTGS** [baileyholl/Terrain-Crystals#23]
In testing this out ore does not seem to spawn from mods (vanilla works great).  It would be nice and much more helpful to skyblock modpack makers to allow for modded ores and biome generation features from mods to generate when using the crystals.  Not sure how possible this is, but thought I'd make the request.

**Comment by baileyholl** [comment]
This is a system that could be implemented however it would require quite a bit of time and thinking about how to spawn modded ores with the correct weight.

**Comment by Flabort** [comment]
Does the 1.11 tag on this mean that we won't be getting this feature for 1.10.2 versions of this mod when it's added?

**Comment by baileyholl** [comment]
Not necessarily, the timing of when content is added varies when I find the time and motivation - 1.10.2 will still be supported as long as it is popular.

**Comment by Deshiba** [comment]
I think most mods will update to 1.11 without a hitch seeing there's not much mechanical differences. Just additions like the observer and a couple of NBT tag changes wouldn't break anything. Not like 1.8 did with rendering and 1.9 did with loot tables. 

There is no real reason to stay on 1.10 for most mods since the underlying engine didn't change. So I suspect most if not all mods/packs will be converted to 1.11 soon™

**Comment by MokahTGS** [comment]
Just a comment on 1.11 vs. 1.10.2:  Most mods (read three times as many) are focusing on 1.10.2.  Unless there is some technical reason to use 1.11, I'd like to see this feature added to 1.10.2 first and then ported to 1.11 once that becomes the dominant version.
